test/sql/types/decimal/test_decimal_from_string.test

```diff
@@ -4,6 +4,26 @@
 #statement ok
 #pragma enable_verification
 
+query I
+select '+1e-1'::DECIMAL(38,3)
```

Comment:
Maybe we should have a bit more tests there, e.g. with exponents other than -1?

Reply:
Ah those are just duplicates of the cases I was last debugging that I forgot to take out.

If you look below there is a monstrosity of 4 nested foreach loops that make up 1400 assertions comparing many variations of pos/neg number, pos/neg exponent, a range of different exponent values and a range of scales